src/Core/System/StateMachine/Aggregation/StateMachineTransition/StateMachineTransitionEntity.php

```diff
@@ -57,12 +58,12 @@ public function setStateMachineId(string $stateMachineId): void
         $this->stateMachineId = $stateMachineId;
     }
 
-    public function getStateMachine(): ?StateMachineStateEntity
+    public function getStateMachine(): ?StateMachineEntity
```

Comment:
the property also has a wrong annotation

Reply:
I foce-push-fixed that in https://github.com/shopware/platform/pull/1893/commits/088d80288d69ca91cada6b6d0c0e58a457d9119d